gusto/linear_solvers.py

```diff
@@ -306,37 +323,42 @@ def __init__(self, state, quadrature_degree=None, solver_parameters=None,
                 state.logger.warning("default quadrature degree most likely not sufficient for this degree element")
             self.quadrature_degree = (5, 5)
 
+        # Turn monitor on for the trace system when running in debug mode
+        if state.output.log_level == DEBUG:
```

Comment:
I just merged a PR that has replaced all the instances of `if state.output.log_level == DEBUG` with `if logger.inEnabledFor(DEBUG)`. I think that in the future we will not have to pass state around so much...

Reply:
Yep I noticed! I'm fixing it locally now as I'm addressing some Jenkins shenanigans